WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -101,6 +101,11 @@
     public static final String ADDRESS_FORMAT_REGEX = "^(https?://(w{3})?|www\\.)";
 
     /**
+     * url that points to wordpress.com purchases
+     */
+    public static final String WORDPRESS_PURCHASES_URL = "https://wordpress.com/purchases";
```

Comment:
I think we should centralize our URL constants rather than scatter them across files. `Constants.java` seems a good place to put this.